.Rhistory

```diff
@@ -0,0 +1,2 @@
+a()
+q()
```

Comment:
Please add .Rhistory to the .gitignore file so the changes there are not pushed

Reply:
thanks, done